.lgtm.yml

```diff
@@ -0,0 +1,17 @@
+extraction:
```

Comment:
since this is a new file, please add comments why this file is need and why we are building protobuf from source.